apps/pollbook/backend/vitest.config.ts

```diff
@@ -7,8 +7,8 @@ export default defineConfig({
     clearMocks: true,
     coverage: {
       thresholds: {
-        lines: 78,
-        branches: 70,
+        lines: -200,
+        branches: -119,
```

Comment:
The actual number of uncovered lines varies from run to run, and this is a number which has passed so far 🤷